### DISCUSSION TITLE
Housekeeping issues

### DISCUSSION THREAD

**Discussion by ikedas** [sympa-community/sympa#1635] [Team Posts]
## Moving place of discussion

Team discussions will be [deprecated in May](https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/).  If no objection, we'd be better to transfer this @sympa-community/everyone to organization discussion.

## Sharing some roles

Currently the following roles are filled by me (@ikedas) alone. If anyone is willing to share each of these roles, please let me know.

*   Listmaster of @sympa.community

*   Owner of en@sympa
    https://lists.sympa.community/info/en

*   Owner of fr@sympa
    https://lists.sympa.community/info/fr

*   Owner of ja@sympa
    https://lists.sympa.community/info/ja

*   Owner of packagers@sympa
    https://lists.sympa.community/info/packagers

**Comment by racke** [comment]
Yes, I'm willing to share these roles.

**Comment by ikedas** [comment]
Thanks @racke.  Let's wait and see if anyone else gets in touch.

**Comment by ikedas** [comment]
Discussion has been moved to the [Discussion on sympa repo](https://github.com/sympa-community/sympa/discussions).

You may have received notices about unread past comments.   Please never mind.

**Comment by ikedas** [comment]
Added.
